shell/src/main/java/alluxio/cli/fs/command/CpCommand.java

```diff
@@ -98,10 +101,18 @@
               + "default is 8MB when copying from local, "
               + "and 64MB when copying to local")
           .build();
+  private static final Option PRESERVE_OPTION =
+      Option.builder("p")
+          .longOpt("preserve")
+          .required(false)
+          .desc("Preserve file attributes when copying files. "
```

Comment:
We can either make this command also preserve TTL, pin state, and replication parameters, or we can just keep the current list and update the description/tests, thoughts?

Reply:
I'd like to keep the functionality as-is since it mirrors what hdfs does. We just need to update the docs (and potentially command name) to make it clear that we're only preserving permissions, not other attributes. We can add additional options for preserving ttl/pin/replication in the future if there's a demand for it.

Reply:
Updated description.